internal/jwt/token.go

```diff
@@ -2,12 +2,15 @@ package jwt
 
 /*
  * This package is meant to find more details about JWT tokens.
- * - Valid: 	Tests the token against IONOS Datacenters API
- * - Claims:	Retrieve Claims payload off a JWT token
- * - Username: 	Given a JWT, retrieve user email using the identity found in the JWT Claims; and using CloudAPI
- * 				User Management API to query the found UUID. Note that the UUID can only be queried if
- * 				its respective user is managed by (or is) the user with that JWT
- *
+ * - Claims:			Retrieve Claims payload off a JWT token
+ * - Headers:			Retrieve Headers off a JWT token
+ * - Username: 			Given a JWT, retrieve user email using the identity found in the JWT Claims; and using CloudAPI
+ * 						User Management API to query the found UUID. Note that the UUID can only be queried if
+ * 						its respective user is managed by (or is) the user with that JWT
+ * - Kid:				Extract TokenId from JWT token Headers
+ * - Uuid:				Extract UserId from JWT token Claims
+ * - ContractNumber:	Extract ContractNumber from JWT token Claims
+ * - Role:				Extract Role from JWT token Claims
```

Comment:
i realize that these comments could be more useful if added right before the respective function? for intellisense.

my bad though, I know that I made these comments here in the first place. I can do this in a separate PR if you want

Reply:
I can change them now, it's no problem.